src/qoi.zig

```diff
@@ -1,3 +1,5 @@
+// updated for zig version 0.12.0
```

Comment:
This isn't zig 0.12.0, please remove this comment

Reply:
oh right, the version is still in progress so it'd be 0.12.0-dev.2543+9eda6ccef, but simplicity sake I'll just say master. I removed the comment.